internal/strongswan/strongswan.go

```diff
@@ -54,20 +54,25 @@ func collectSasStats(configs []map[string]vici.IKEConf, sas []map[string]vici.Ik
 		}
 	}
 
-	log.Infof("Sas: %d", len(sas))
```

Comment:
Removing this without replacing it with something else will make us unable to see `len(sas) == 0`. Not sure if thats a problem

Reply:
There will be a log for every SA so this will still be visible. 👍